spec/classes/repository_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+describe 'bareos::repository' do
+  on_supported_os.each do |os, facts|
+    context "on #{os} " do
+      let :facts do
+        facts
+      end
+
+      context 'with default values for all parameters' do
+        it { is_expected.to compile }
```

Comment:
can you update this so the whole catalog is verified?
```suggestion
        it { is_expected.to compile.with_all_deps }
```

Reply:
done